packages/cloud_firestore/cloud_firestore/lib/src/firestore_message_codec.dart

```diff
@@ -51,7 +51,7 @@ class FirestoreMessageCodec extends StandardMessageCodec {
       buffer.putFloat64(value.longitude);
     } else if (value is DocumentReference) {
       buffer.putUint8(_kDocumentReference);
-      final List<int> appName = utf8.encoder.convert(value.firestore.app.name);
+      final List<int> appName = utf8.encoder.convert(value.firestore.appName());
```

Comment:
I think you forgot to delete this file, so you use the copy from the platform_interface package?